### PR TITLE
XIVY-2250 extend Dynamic Dialog Demo

### PR DESCRIPTION
**Pull request by ivy-tgr**

### CODE REVIEW

**Comment by ivy-tgr** [comment]
As discussed with @ivy-cst I have extended the demo to show that you can access the data from the dialog and also access to the Logic.

File: html-dialog/html-dialog-demos/src_hd/ch/ivyteam/htmldialog/demo/output/DynamicDialogDemo/FinalDlg.xhtml [hunk 1]
```diff
@@ -13,6 +13,7 @@
         <p:outputLabel>This is the called Dynamic Final Dialog</p:outputLabel><br/><br/>
           
           <p:commandButton style="width:180px" id="closeDialog" actionListener="#{dynamicDialog.close('FinalDlg')}" value="Close Final Dialog" icon="ui-icon-check" update="finalDialogForm"/>
+          <p:outputLabel value="#{data.data}"></p:outputLabel>
```
Comment (ivy-rew) [inline html-dialog/html-dialog-demos/src_hd/ch/ivyteam/htmldialog/demo/output/DynamicDialogDemo/FinalDlg.xhtml:16]: data.data looks bad, I'd use something like "data.message"

File: html-dialog/html-dialog-demos/src_hd/ch/ivyteam/htmldialog/demo/output/DynamicDialogDemo/DynamicDialogDemoProcess.p.json [hunk 2]
```diff
@@ -46,5 +46,31 @@
         "at" : { "x" : 224, "y" : 160 },
         "labelOffset" : { "x" : 10, "y" : 30 }
       }
+    }, {
+      "id" : "f6",
+      "type" : "HtmlDialogEventStart",
+      "name" : "event",
```
Comment (ivy-rew) [inline html-dialog/html-dialog-demos/src_hd/ch/ivyteam/htmldialog/demo/output/DynamicDialogDemo/DynamicDialogDemoProcess.p.json:52]: also here; I'd use something less technical as name.

**Review by ivy-rew — COMMENTED** [review]
cool that this feature works; make this dynamic dialog stuff more usable.

**Pull request merged**